scripts/core/editor3/components/handlePastedText.ts

```diff
@@ -46,7 +46,9 @@ function pasteContentFromOpenEditor(
                     internalClipboard.forEach((b) => blocksArray.push(b));
                     const contentState = ContentState.createFromBlockArray(blocksArray);
 
-                    return insertContentInState(editorState, contentState, onChange, editorFormat);
+                    const editorWithContent = insertContentInState(editorState, contentState, editorFormat);
+
+                    return 'handled';
```

Comment:
If you are returning 'handled', is there a point to do the operations from line 44-49?

Reply:
Sorry, I thought you were returning 'not-handled'. Anyway, it doesn't seem `const editorWithContent` is used anywhere. If it isn't, you don't need the operations?

Reply:
Forgot to call onChange again, fixed it on latest commit